packages/pn-commons/src/components/NotificationDetail/NotificationDetailDocuments.tsx

```diff
@@ -40,9 +40,13 @@ const NotificationDetailDocuments: React.FC<Props> = (
 ) => {
   const mapOtherDocuments = (documents: Array<NotificationDetailDocument>) =>
     documents.map((d) => {
+      const recipient =
+        documents.length > 1 && d.documentType === 'AAR' && d.recipient
```

Comment:
I think you are using documents.length > 1 to check if the notification is multi recipient. It is not completely correct. You should use the check on the recipients array